java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -329,14 +332,24 @@ protected Object[] handleArrayOrNullResponse(Response response) throws RedisExce
 
     /**
      * @param response A Protobuf response
-     * @return A map of <code>String</code> to <code>V</code>.
      * @param <V> Value type.
+     * @return A map of <code>String</code> to <code>V</code>.
```

Comment:
nit:  you can revert this.  
`V` is part of the return (Map Typing) and it makes sense to document this after the `@return`.